### DISCUSSION TITLE
[Request] Add channel blocking feature

### DISCUSSION THREAD

**Discussion by outoffuel** [qnblackcat/uYouPlus#355] [Ideas]
Please add channel blocking feature like [Blocktube](https://github.com/amitbl/blocktube)

**Comment by WoodpeckerBaby** [comment]
This would be a god send. I use this on desktop, and I hope there is a way to use something this in app as well. 

https://chrome.google.com/webstore/detail/channel-blocker/nfkmalbckemmklibjddenhnofgnfcdfp
